### PR TITLE
100 % mutation score

### PR DESCRIPTION
**Pull request by kukulich**

### CODE REVIEW

File: src/Reflection/ReflectionProperty.php
Comment (Ocramius) [inline src/Reflection/ReflectionProperty.php]: Related: https://github.com/sebastianbergmann/php-code-coverage/pull/969 ?

Comment (kukulich) [reply src/Reflection/ReflectionProperty.php]: Yes, it looks it's related. So this PR can probably wait for the release :)

Comment (Ocramius) [reply src/Reflection/ReflectionProperty.php]: Probably worth doing that, ya :+1:

Comment (Slamdunk) [reply src/Reflection/ReflectionProperty.php]: Butfix released, a rebase now should give 100% MSI

**Comment by kukulich** [comment]
And again one new mutant remaining.

**Comment by kukulich** [comment]
I've just added ignore for this last mutant. I don't know what I should change and how to test it. And I really want to merge this PR and unblock dependencies upgrades 😄 

```
Warning: Escaped Mutant for Mutator "FalseValue":

--- Original
+++ New
@@ @@
     public function stream_open($path, $mode, $options, &$opened_path) : bool
     {
         self::$autoloadLocatedFile = $path;
-        return false;
+        return true;
```

**Review by Ocramius — APPROVED** [review]
Thanks @kukulich !

**Pull request merged**
